spec/features/solicitor/dashboard_spec.rb

```diff
@@ -8,14 +8,16 @@
     accepted_defence_request = create(
       :defence_request,
       :accepted,
-      solicitor_uid: solicitor_user.uid
+      solicitor_uid: solicitor_user.uid,
+      organisation_uid: solicitor_user.organisation_uids.first,
+      offences: "Arson"
     )
-    not_accepted_defence_request = create :defence_request
+    not_accepted_defence_request = create :defence_request, offences: "Extortion"
 
-    login_with_role "solicitor", solicitor_user.uid
+    login_with_user solicitor_user
```

Comment:
Any reason for changing this method name? I'm fine with passing less parameters around, but if feels the original name provided more context.